user_service/src/controllers/UserController.ts

```diff
@@ -43,4 +43,56 @@ export const delUserProfile = async (req: any, res: any) => {
             error: error,
             message: 'User not found.'})
     }
+}
+
+
+//@desc     get user's answered questions
+//@route    GET /api/users/:userId/questions
+//@access   authenticated users
+
+export async function getUserQuestions(req: any, res: any) {
+  try {
+    const userId = parseInt(req.params.userId, 10); // Parse userId as an integer
```

Comment:
![image](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g10/assets/72195240/59c82404-f9d1-480a-b62a-3084a70d68ed)

-----------

![image](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g10/assets/72195240/9c30107f-6698-4bba-8632-5a05b2159ea2)

routes doesnt match. one uses `:userID` but the actual implementation uses `:id`

Reply:
fixed in [Removed unecessary question Data and minor changes to the usercontrol…](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g10/pull/43/commits/d5fade30cb612fa6619b082c1db541d073587d4b)